api/src/main/java/com/datastrato/gravitino/rel/expressions/literals/Literals.java

```diff
@@ -33,6 +66,66 @@ public static LiteralImpl<Integer> integer(Integer value) {
     return of(value, Types.IntegerType.get());
   }
 
+  /**
+   * Creates a long type literal with the given value.
+   *
+   * @param value the long literal value
+   * @return a new {@link Literal} instance
+   */
+  public static LiteralImpl<Long> longLiteral(Long value) {
+    return of(value, Types.LongType.get());
+  }
+
+  /**
+   * Creates a float type literal with the given value.
+   *
+   * @param value the float literal value
+   * @return a new {@link Literal} instance
+   */
+  public static LiteralImpl<Float> floatLiteral(Float value) {
+    return of(value, Types.FloatType.get());
+  }
+
+  /**
+   * Creates a double type literal with the given value.
+   *
+   * @param value the double literal value
+   * @return a new {@link Literal} instance
+   */
+  public static LiteralImpl<Double> doubleLiteral(Double value) {
+    return of(value, Types.DoubleType.get());
+  }
+
+  /**
+   * Creates a date type literal with the given value.
+   *
+   * @param value the date literal value
+   * @return a new {@link Literal} instance
+   */
+  public static LiteralImpl<LocalDate> date(LocalDate value) {
+    return of(value, Types.DateType.get());
+  }
+
+  /**
+   * Creates a time type literal with the given value.
+   *
+   * @param value the time literal value
+   * @return a new {@link Literal} instance
+   */
+  public static LiteralImpl<LocalTime> time(LocalTime value) {
+    return of(value, Types.TimeType.get());
+  }
+
+  /**
+   * Creates a timestamp type literal with the given value.
+   *
+   * @param value the timestamp literal value
+   * @return a new {@link Literal} instance
+   */
+  public static LiteralImpl<LocalDateTime> timestamp(LocalDateTime value) {
```

Comment:
Why here use `LocalDate`, `LocalTime` and `LocalDateTime`?

Reply:
Because they are both commonly used objects in the JDK to represent date and time.